{"stats": {"errors": 1, "failures": 0, "passes": 0, "skipped": 0, "encoding": "UTF-8", "total": 1}, "results": [{"classname": "index_test:TxTest", "name": "test_parse_outputs", "time": 0.0003097057342529297, "type": "error", "errtype": "builtins.TypeError", "message": "'bytes' object cannot be interpreted as an integer", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/pb-tester-ccdle12/tx-outputs-jsong-pb-sf-072618/test/index_test.py\", line 10, in test_parse_outputs\n    tx = Tx.parse(stream)\n  File \"/home/pb-tester-ccdle12/tx-outputs-jsong-pb-sf-072618/index.ipynb\", line 49, in parse\n    \"        num_inputs = read_varint(s)\\n\",\n  File \"/home/pb-tester-ccdle12/tx-outputs-jsong-pb-sf-072618/index.ipynb\", line 66, in parse\n    \"\\n\",\n  File \"/home/pb-tester-ccdle12/tx-outputs-jsong-pb-sf-072618/helper.py\", line 29, in little_endian_to_int\n    return int.from_bytes(b, 'little')\nTypeError: 'bytes' object cannot be interpreted as an integer\n"}]}